charts/tempo-distributed/values.yaml

```diff
@@ -419,7 +426,7 @@ querier:
 queryFrontend:
   query:
     # -- Required for grafana version <7.5 for compatibility with jaeger-ui. Doesn't work on ARM arch
-    enabled: true
+    enabled: false
```

Comment:
Not sure if this change should be in the same PR, seems unrelated to all the others changes (unless I am missing something)

Reply:
it's incompatible with readonlyrootfilesystem and @joe-elliott suggested disabling it since it was outdated

Reply:
> Well we can just rename it to "security hardening of tempo-distributed", maybe.
> Initial adding of /var/tempo emptyDir was done to make it compatible with security best practices, so, IMO, makes sense to add best practices to same PR. (especially since other charts, like loki and mimir, already comes with preconfigured securityContext and etc)

Was just discussing it with Elliot ; he said the same thing :D 
How about my 2 suggestions ?